kubernetes/zeus/kube-lego/configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-lego
+  namespace: kube-lego
+data:
+  # modify this to specify your address
+  lego.email: "ops@sentry.io"
```

Comment:
I assume not possible to actually make this configurable? Would be nice for this to not blast our mailing list if someone else decides to run this themselves without editing it.

Reply:
i think the way this becomes configurable would be through templates and Helm-- outside of that you'd set this via the CLI when you create the recipe but thats a pita.